src/apolloServer.ts

```diff
@@ -13,23 +13,47 @@ interface UserInput {
   birthdate: string;
 }
 
+function getUsers() {
+  return AppDataSource.manager.getRepository('user').find();
+}
+
+async function validateInputs(args: { input: UserInput }) {
+  const passwordRegex = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{6,}$/;
+
+  if (!passwordRegex.test(args.input.password)) {
+    throw new Error('Password must be at least 6 characters long, have at least 1 letter and 1 digit.');
+  }
+
+  const existentUserWithEmail = await AppDataSource.manager
```

Comment:
```suggestion
  const existentUser = await AppDataSource.manager
```

Assim já fica claro o suficiente, não?

Reply:
Modificado.